src/hmem_cuda.c

```diff
@@ -630,20 +521,20 @@ int cuda_hmem_init(void)
 	param_value = 1;
 	ret = fi_param_get_bool(NULL, "hmem_cuda_enable_xfer", &param_value);
 	if (ret) {
-		hmem_cuda_xfer_setting = CUDA_XFER_UNSPECIFIED;
+		cuda_attr.xfer_setting = CUDA_XFER_UNSPECIFIED;
 	} else {
-		hmem_cuda_xfer_setting = (param_value > 0) ? CUDA_XFER_ENABLED : CUDA_XFER_DISABLED;
+		cuda_attr.xfer_setting = (param_value > 0) ? CUDA_XFER_ENABLED : CUDA_XFER_DISABLED;
```

Comment:
while you're here, can you fix the if/else to remove the {}

Reply:
Ack. This can actually be simplified a bit since I initialize this to `CUDA_XFER_UNSPECIFIED` anyway

Reply:
Actually we should deprecate xfer_setting because we introduced FI_OPT_CUDA_API_PERMITTED. 

Can we just remove it (maybe a separate patch)?